### PR TITLE
react_goods-selector

### PR DESCRIPTION
**Pull request by georgiyevv**
[DEMO LINK](https://svyatoslavPy.github.io/react_goods-selector-js/)

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Make sure to deploy your solution

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
Demo link still doesn't work
image

File: src/App.jsx [hunk 1]
```diff
@@ -14,57 +16,72 @@ export const goods = [
...
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      {selectedGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button
+            onClick={() => clearSelectedGood()}
```
Comment (GUSILLUS) [inline src/App.jsx:36]: Consider writing like this
```suggestion
            onClick={clearSelectedGood}
```

File: src/App.jsx [hunk 2]
```diff
@@ -14,57 +16,72 @@ export const goods = [
...
+                    onClick={() => selectGood(good)}
+                    data-cy="AddButton"
+                    type="button"
+                    className="button"
+                  >
+                    +
+                  </button>
+                ) : (
+                  <button
+                    onClick={() => clearSelectedGood()}
```
Comment (GUSILLUS) [inline src/App.jsx:68]: same

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
GJ! And try to fix lint problem

**Review by GUSILLUS — APPROVED** [review]
Approve!

**Pull request closed without merging**